webbpsf/webbpsf_core.py

```diff
@@ -920,6 +921,87 @@ def load_was_opd(self, inputWasOpd, size=1024, save=False, filename='new_was_opd
+
+        Parameters
+        ----------
+        all_detectors: bool
+            If True, run all detectors for the instrument. If False, run for the detector set in
+            the instance. Default is True
+        use_detsampled_psf: bool
+            If True, the grid of PSFs returned will be detector sampled (made by binning down the
+            oversampled PSF). If False, the PSFs will be oversampled by the factor defined by the
+            oversample/detector_oversample/fft_oversample keywords. Default is False.
+        num_psfs: int
+            The total number of fiducial PSFs to be created and saved in the files. This
+            number must be a square number. Default is 16.
+            E.g. num_psfs = 16 will create a 4x4 grid of fiducial PSFs.
+        single_psf_centered: bool
+            If num_psfs is set to 1, this defines where that psf is located. If True it will be the
+            center of the detector, if False it will be the location defined in the WebbPSF
+            attribute detector_position (reminder - detector_position is (x,y)). Default is True
+        save: bool
+            True/False boolean if you want to save your file. Default is False.
+        outfile: str
+            If "save" keyword is set to True, your current file will be saved under
+            "{outfile}_det_filt.fits". Default of None will save it in the current
+            directory as: instr_det_filt_fovp#_samp#_npsf#.fits
+        overwrite : bool
+            True/False boolean to overwrite the output file if it already exists. Default
+            is True.
+        **kwargs:
+            Add any extra arguments to the WebbPSF calc_psf() method call. This includes
+            the following kwargs (and their default values): source(=None), nlambda(=None),
+            monochromatic(=None), fov_arcsec(=None), fov_pixels(=101), oversample(=4),
+            detector_oversample(=None), fft_oversample(=None), normalize(='first'),
+            add_distortion(=True), and crop_psf(=True)
+
+        Returns
+        -------
+        grid : fits.HDUList or list
+            Returns and/or saves 3D fits.HDUlist object (1 per instrument, detector, and filter)
+            or a list of fits.HDUlist objects if more than one detector is specified
+
+        Use
+        ----
+        nir = webbpsf.NIRCam()
+        nir.filter = "F090W"
+        nir.psf_grid(all_detectors=True, num_psfs=4)
+
+        nir = webbpsf.NIRCam()
+        nir.filter = "F090W"
+        nir.detector = "NRCA2"
+        nir.psf_grid(all_detectors=False, oversample=5, fov_pixels=101)
+
+        """
+
+        # Keywords that could be set before the method call
+        filters = self.filter
+
+        if all_detectors is True:
+            detectors = "all"
+        else:
+            detectors = self.detector
+
+        if single_psf_centered is True:
+            psf_location = (int(self._detector_npixels / 2), int(self._detector_npixels / 2))  # center point
```

Comment:
In fact the center location has to be computed as `(npixels-1)/2`. (You can think about the case of npixels=2 and npixels=3, and you should be able to convince yourself why this is the case, for Python's 0-based array indexing...)

```suggestion
            psf_location = (int((self._detector_npixels-1) / 2), int((self._detector_npixels-1) / 2))  # center point
```